src/openforms/forms/migrations/0088_enable_mutiple_registration_backends.py

```diff
@@ -0,0 +1,111 @@
+# Generated by Django 3.2.20 on 2023-07-18 14:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+import openforms.registrations.fields
+from openforms.registrations.registry import register
+
+
+def forward(apps, _):
+    Form = apps.get_model("forms", "Form")
+    FormRegistrationBackend = apps.get_model("forms", "FormRegistrationBackend")
+    for form in Form.objects.exclude(
+        models.Q(registration_backend="") | models.Q(registration_backend__isnull=True)
+    ):
```

Comment:
I think it's possible to have registration backends without options? Like ZGW with default options? It doesn't look particularly realistic, *but* it could also be partial configuration and I wonder if there's any harm in copying those over?

Reply:
agreed that it's pretty much only possible directly in the admin/ORM, the API doesn't seem to allow it.